schema/types/base/id.pb.go

```diff
@@ -0,0 +1,150 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.17.3
+// source: proto/persistence/schema/types/base/v1beta1/id.proto
+
+package base
+
+import (
+	_ "github.com/regen-network/cosmos-proto"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type Id struct {
```

Comment:
does it have to be exported?

Reply:
we should use the id struct have it convert to this struct

Reply:
It does not have to be imported. It is generated in caps when pb.go file is generated.